RetailCoder.VBE/API/VBA/ParserState.cs

```diff
@@ -64,14 +65,15 @@ public void Initialize(Microsoft.Vbe.Interop.VBE vbe)
 
             _vbe = new VBE(vbe);
             var declarationFinderFactory = new ConcurrentlyConstructedDeclarationFinderFactory();
-            _state = new RubberduckParserState(null, declarationFinderFactory);
+            var projectRepository = new ProjectsRepository(_vbe);
```

Comment:
Why is this `new`'d? Should we be using DI here too?

Reply:
This is the COM accessible API. Everything here is newed up.